apps/nextjs-website/src/app/auth/sign-up/page.tsx

```diff
@@ -0,0 +1,175 @@
+      return result;
+    } else {
+      router.replace(
+        `/auth/sign-up?email=${username}&step=${SignUpSteps.CONFIRM_SIGN_UP}`
+      );
+      setSignUpStep(SignUpSteps.CONFIRM_SIGN_UP);
+      return !!result.user;
+    }
+  }, [
+    company,
+    firstName,
+    lastName,
+    mailinglistAccepted,
+    password,
+    role,
+    router,
+    username,
+  ]);
+
+  const onBackStep = useCallback(() => {
+    router.replace(
+      `/auth/sign-up?email=${username}&step=${SignUpSteps.SIGN_UP}`
+    );
+    setSignUpStep(SignUpSteps.SIGN_UP);
+    return null;
+  }, [router, username]);
+
+  const onResendEmail = useCallback(async () => {
+    await Auth.resendSignUp(username);
+    return null;
+  }, [username]);
+
+  if (environment === 'prod') {
+    return <PageNotFound />;
+  }
+
+  return (
+    <>
+      <Box
+        sx={{
+          display: 'flex',
+          alignItems: 'center',
+          justifyContent: 'center',
+          backgroundImage: 'url(/images/hero.jpg)',
+          backgroundRepeat: 'no-repeat',
+          backgroundSize: 'cover',
+          backgroundPosition: 'bottom right',
+        }}
+      >
+        <Grid
+          container
+          spacing={isSmallScreen ? 0 : 1}
+          alignItems={isSmallScreen ? 'center' : 'flex-start'}
+          justifyContent='center'
+          direction={isSmallScreen ? 'column-reverse' : 'row'}
+          mx={isSmallScreen ? 3 : 20}
+          my={isSmallScreen ? 3 : 15}
+        >
+          <Grid item xs={isSmallScreen ? 1 : 5}>
+            <Typography variant='h6' mb={4} mt={isSmallScreen ? 10 : 0}>
+              {signUp.whyCreateAccount}
+            </Typography>
+            {signUp.advantages.map((advantage, index) => {
+              return (
+                <CheckItem
+                  key={index}
+                  title={`Vantaggio ${index}`}
+                  description={advantage}
+                />
+              );
+            })}
+          </Grid>
+          <Grid item xs={isSmallScreen ? 1 : 5}>
+            {signUpStep === SignUpSteps.SIGN_UP && (
+              <SignUpForm
+                username={username}
+                setUsername={setUsername}
+                password={password}
+                setPassword={setPassword}
+                confirmPassword={confirmPassword}
+                setConfirmPassword={setConfirmPassword}
+                firstName={firstName}
+                setFirstName={setFirstName}
+                lastName={lastName}
+                setLastName={setLastName}
+                company={company}
+                setCompany={setCompany}
+                role={role}
+                setRole={setRole}
+                mailinglistAccepted={mailinglistAccepted}
+                setMailinglistAccepted={setMailinglistAccepted}
+                onSignUp={onSignUp}
+              />
+            )}
+            {signUpStep === SignUpSteps.CONFIRM_SIGN_UP && (
+              <ConfirmSignUp
```

Comment:
When I'm on the confirmation page, if I refresh the page and click the "Reinvia e-mail" link I get an error

Reply:
There's a bug with the email aliases (those with a `+` sign), it doesn't happen with regular emails, should we correct for those email or we want to let users set up multiple accounts with the same email even in production?

Reply:
@MikeAtUqido IMHO it should be fixed. I think the best option, at the moment, is to create an activity explaining the problem and then add the link of the activity here.
can you do it, please?

Reply:
Here's the activity: [DEV-1114](https://pagopa.atlassian.net/browse/DEV-1114)